tools/preflight/preflight.go

```diff
@@ -757,6 +759,8 @@ func (o *Run) createSourcePodFromPVC(ctx context.Context, nameSuffix, pvcName st
 	o.Logger.Infof("Waiting for source pod - %s to become ready\n", srcPod.GetName())
 	err = waitUntilPodCondition(ctx, waitOptions)
 	if err != nil {
+		srcPodYaml := objToYAML(srcPod)
+		o.Logger.Warnf("pod yaml: \n%s", string(srcPodYaml))
```

Comment:
Log statements should be consistent across the given application. Here, the log stmt does not convey much info.
Same situation is handled in case of DNS pod. Check that. The log statement here should be on the similar lines.